pyuoi/linear_model/logistic.py

```diff
@@ -0,0 +1,54 @@
+from .base import AbstractUoILinearRegressor
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import l1_min_c
+
+import numpy as np
+
+
+class UoI_L1Logistic(AbstractUoILinearClassifier):
+
+    def __init__(self, n_boots_sel=48, n_boots_est=48, selection_frac=0.9,
```

Comment:
Should also include the `multi_class` keyword to support multinomial logistic regression.

Reply:
Err, maybe not since I'm not sure how to do the UoI parts with multinomial.